src/libhdf5.settings.in

```diff
@@ -76,6 +76,7 @@ Dimension scales w/ new references: @DIMENSION_SCALES_WITH_NEW_REF@
                   Build HDF5 Tests: @HDF5_TESTS@
                   Build HDF5 Tools: @HDF5_TOOLS@
                    Build GIF Tools: @HDF5_HL_GIF_TOOLS@
+                           Threads: @THREADS@
```

Comment:
Need to update the config/cmake/libhdf5.settings.cmake.in file

Also should we add an option variable to hdf5-config.cmake.in file.

Reply:
Done, thanks for catching these